From 9b1e1db723ddc9d0838a8067cabfbece6c5d0071 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Thu, 21 Feb 2019 08:11:06 +0100
Subject: [PATCH 002/386] arm64: dts: marvell: 913x: complement CP2 SD
 description

Correct vqmmc-supply regulator GPIO pin number and add
card-detect GPIO to the CON2 slot.

Change-Id: I0e045cec015f0793dc4569358038368d51989d2c
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/4867
Reviewed-by: Grzegorz Jaszczyk <Grzegorz.Jaszczyk@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Tested-by: sa_ip-sw-jenkins
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9132-db.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9132-db.dtsi b/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
index 6a23582230c9..e6c43dd7c5cc 100644
--- a/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
@@ -58,7 +58,7 @@
 		regulator-name = "cp2_sd_vcc";
 		regulator-min-microvolt = <1800000>;
 		regulator-max-microvolt = <3300000>;
-		gpios = <&cp2_gpio2 16 GPIO_ACTIVE_HIGH>;
+		gpios = <&cp2_gpio2 17 GPIO_ACTIVE_HIGH>;
 		states = <1800000 0x1 3300000 0x0>;
 	};
 
@@ -119,6 +119,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&cp2_sdhci_pins>;
 	bus-width = <4>;
+	cd-gpios = <&cp2_gpio2 23 GPIO_ACTIVE_LOW>;
 	vqmmc-supply = <&cp2_reg_sd_vccq>;
 };
 
-- 
2.17.1

